project/PACKAGERS.md

```diff
@@ -44,8 +44,7 @@ need to package Docker your way, without denaturing it in the process.
 To build Docker, you will need the following:
 
 * A recent version of Git and Mercurial
-* Go version 1.4 or later (Go version 1.5 or later required for hardware signing
-  support in Docker Content Trust)
+* Go version 1.6.2 or later
```

Comment:
Yeah, I think so -- the security fix was in 1.6.1, but if we're compatible
with 1.6, then I think the specific minor version is really the
responsibility of the reader to choose (or to be stuck with, as the case
may be).